atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/backup/CqlCluster.java

```diff
@@ -97,14 +100,75 @@ public static CqlCluster create(CassandraKeyValueServiceConfig config) {
         }
     }
 
+    public KeyedStream<String, Map<InetSocketAddress, Set<TokenRange>>> getTransactionsTableRangesForRepair(
```

Comment:
I'm personally not much of a fan of passing around streams unless it's really performance sensitive. I get that you don't want to page through the stream multiple times, but are these really going to be that large? It appears bounded by keyspace, which in the absolute worst case shouldn't be more than say 400, and more often 100-200, where this is a trivial operation.

My aversion stems from the fact that it's much easier to screw up a stream than something else. Maybe that's just me overreacting, but if you don't _need_ to do it, I'd advise against it.